scripts/seed.py

```diff
@@ -54,6 +67,9 @@ def create_challenge_host_team(user):
 
 
 def create_challenges(number_of_challenges=3, host_team=None):
+    """
+    Creates past challenge, on-going challenge, upcoming challenge.
```

Comment:
Replace with `Creates past challenge, on-going challenge and upcoming challenge.`